gammapy/background/models.py

```diff
@@ -71,3 +81,775 @@ def evaluate(self, x, y):
+        hdu.header['CRPIX1'] = wcs_header['CRPIX1']
+        hdu.header['CRPIX2'] = wcs_header['CRPIX2']
+        hdu.header['CRVAL1'] = wcs_header['CRVAL1']
+        hdu.header['CRVAL2'] = wcs_header['CRVAL2']
+
+        return hdu
+
+    def write(self, outfile, format='bin_table', write_kwargs=None):
+        """Write cube background model to fits file.
+
+        Several output formats are accepted, depending on the value
+        of the `format` parameter:
+
+        * bin_table (default and preferred format): `~astropy.io.fits.BinTableHDU`
+        * image (alternative format): `~astropy.io.fits.ImageHDU`
+
+        Depending on the value of the `format` parameter, this
+        method calls either `~astropy.io.fits.BinTableHDU.writeto` or
+        `~astropy.io.fits.ImageHDU.writeto`, forwarding the
+        `write_kwargs` arguments.
+
+        Parameters
+        ----------
+        outfile : `~string`
+            name of file to write
+        format : `~string`, optional
+            format of the bg cube to write
+        write_kwargs : `~dict`, optional
+            extra arguments for the corresponding `io.fits` `writeto` method
+        """
+        if write_kwargs is None:
+            write_kwargs = dict()
+
+        if format == 'bin_table':
+            self.to_fits_bin_table().writeto(outfile, **write_kwargs)
+        elif format == 'image':
+            self.to_fits_image().writeto(outfile, **write_kwargs)
+        else:
+            raise ValueError("Got format {}, which is not understood."
+                             .format(format))
+
+    @property
+    def image_extent(self):
+        """Image extent `(x_lo, x_hi, y_lo, y_hi)`.
+
+        Returns
+        -------
+        im_extent : `~astropy.coordinates.Angle`
+            array of bins with the image extent
+        """
+        bx = self.detx_bins.to('degree')
+        by = self.dety_bins.to('degree')
+        return Angle([bx[0], bx[-1], by[0], by[-1]])
+
+    @property
+    def spectrum_extent(self):
+        """Spectrum extent `(e_lo, e_hi)`.
+
+        Returns
+        -------
+        spec_extent : `~astropy.units.Quantity`
+            array of bins with the spectrum extent
+        """
+        b = self.energy_bins.to('TeV')
+        return Quantity([b[0], b[-1]])
+
+    @property
+    def image_bin_centers(self):
+        """Image bin centers `(x, y)`.
+
+        Returns
+        -------
+        det_edges_centers : `~astropy.coordinates.Angle`
+            array of bins with the image bin centers [[x_centers], [y_centers]]
+        """
+        detx_edges_low = self.detx_bins[:-1]
+        detx_edges_high = self.detx_bins[1:]
+        detx_edges_centers = (detx_edges_low + detx_edges_high)/2.
+        dety_edges_low = self.dety_bins[:-1]
+        dety_edges_high = self.dety_bins[1:]
+        dety_edges_centers = (dety_edges_low + dety_edges_high)/2.
+        return Angle([detx_edges_centers, dety_edges_centers])
```

Comment:
Since you were explicity about returning `degree` above, maybe add `degree` to the `Angle` constructor call here, too?
(And mention degree in the docstring)